Source/svg/SVGParser.swift

```diff
@@ -1393,6 +1393,9 @@ open class SVGParser {
                     let transform = transformBoundingBoxLocus(respectiveLocus: userSpaceLocus.locus, absoluteLocus: locus)
                     return TransformedLocus(locus: userSpaceLocus.locus, transform: transform)
                 }
+                if let userSpaceTransform = userSpaceLocus.transform, userSpaceTransform != Transform.identity {
```

Comment:
I added a check here to only return a `TransformedLocus` if the transform is not the identity matrix.